taurus/client/tests/test_json.py

```diff
@@ -78,8 +78,9 @@ def test_thin_dumps():
     assert isinstance(thin_copy.spec, LinkByUID)
     assert thin_copy.spec.id == meas_spec.uids['my_scope']
 
-    with pytest.raises(TypeError):
-        thin_dumps(LinkByUID('scope', 'id'))
+    # Check that LinkByUID objects are correctly converted their JSON equivalent
+    expected_json = '{"id": "my_id", "scope": "scope", "type": "link_by_uid"}'
+    assert thin_dumps(LinkByUID('scope', 'my_id')) == expected_json
```

Comment:
Does thin_dumps enforce/provide key ordering?  This seems like it could cause spurious test failures if the keys weren't output alphabetically.

Reply:
That's a good question, I'll look into this.

Reply:
I can see that thin_dumps uses sort_keys=True when using json.dumps.  These are the same parameters as used by the dumps function in json_encoder.py.  I think it's safe to assume that key ordering is unchanged.